test/Microsoft.ML.Tests/Scenarios/GetColumnTests.cs

```diff
@@ -1,121 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-using System.Linq;
-using System.Reflection;
-using Microsoft.ML.Data;
-using Microsoft.ML.RunTests;
-using Microsoft.ML.Runtime;
-using Microsoft.ML.StaticPipe;
-using Microsoft.ML.TestFramework;
-using Xunit;
-using Xunit.Abstractions;
-
-namespace Microsoft.ML.Tests.Scenarios
-{
-    public sealed class GetColumnTests : BaseTestClass
-    {
-        public GetColumnTests(ITestOutputHelper output) : base(output)
-        {
-        }
-
-        [Fact]
-        public void TestGetColumn()
-        {
-            var path = GetDataPath(TestDatasets.breastCancer.trainFilename);
-            var env = new MLContext();
-            var data = TextLoaderStatic.CreateLoader(env, ctx => (
-                floatScalar: ctx.LoadFloat(1),
-                floatVector: ctx.LoadFloat(2, 6),
-                stringScalar: ctx.LoadText(4),
-                stringVector: ctx.LoadText(5, 7)
-            )).Load(path);
-
-            var enum1 = data.AsDynamic.GetColumn<float>(data.AsDynamic.Schema["floatScalar"]).ToArray();
```

Comment:
Do we still want these "dynamic" GetColumn tests?

Reply:
Yes we should keep these, you are right, I'll just remove the part that's static api only.

---
In reply to: [304498224](https://github.com/dotnet/machinelearning/pull/4009#discussion_r304498224) [](ancestors = 304498224)